build/verify-features.ts

```diff
@@ -10,11 +10,11 @@ const featuresInReadme = getFeaturesMeta();
 const errors: string[] = [];
 
 for (let fileName of featuresDirContents) {
-	if (['css', 'index.tsx'].includes(fileName) || fileName.includes('rgh')) {
+	if (fileName === 'index.tsx' || fileName.includes('.css') || fileName.includes('rgh')) {
```

Comment:
Exactly why I deleted the comment :)

Reply:
Ahh I pressed resolve and the comment disappeared - now I know why 😅

Reply:
```suggestion
	if (fileName === 'index.tsx' || fileName.endsWith('.css') || fileName.startsWith('rgh')) {
```